Makefile

```diff
@@ -36,11 +36,15 @@ tini/src:
 .PHONY: build
 build: tini/src | $(clean) $(builder)
 	mkdir -p $(PWD)/build/distributions
-	$(DOCKER_RUN) -v $(PWD):/src -v $(PWD)/build/distributions:/dist -u $(UID):$(GID) \
+	$(DOCKER_RUN) -v $(PWD):$(PWD) -u $(UID):$(GID) \
```

Comment:
this assumes the current local directory is in the `GOPATH`. I'd rather we don't assume that and put things in the GOPATH the docker image expects.

Reply:
This assumes that the current working directory is the path to the Titus executor project, and that the path to the titus executor project is in the GOPATH. If the GOPATH inside-and-outside of the container are different, Golang gets confused. Part of this is to do with caching, and part of it, I do not understand.

Reply:
This makes assumptions about the local dev environment of whoever is running these `make` tasks. Everything runs in a container, no need to have a `GOPATH` configured or a strict requirement on the filesystem layout for the project. One can't just checkout the project and run the `make` targets anymore, there needs to be a `GOPATH`, etc, etc, even when the project isn't being compiled outside of the builder containers.

Unnecessary additional complexity on whoever tries to build the project locally without a Go dev environment IMHO.